packages/docs/src/examples/sizing/height.vue

```diff
@@ -0,0 +1,11 @@
+<template>
```

Comment:
example seems incomplete? (same width.vue)

Reply:
I'm not doing examples until the docs are in v3.